### ISSUE TITLE
Feature request: Blocklist Synonyms 

### ISSUE THREAD

**Issue by oliverjessner** [wit-ai/wit#1860]
**Do you want to request a *feature* about wit?**
My request is a duplicate of:  https://github.com/wit-ai/wit/issues/855
The problem with implementing it in duckling is, that we don't have any Haskel programmers 
at my company. I don't even know one person having experience in Haskel.
Maybe the wit team or some else has some spare time to to add this feature.
Thanks in advance.

**What is the expected behavior?**
A simple input element to enter strings which are blocked by wit.

**Comment by aforaleka** [comment]
Hi @oliverjessner, if this is related to parsing date time, I suggest opening a Github issue in our [duckling](https://github.com/facebook/duckling/issues) repo and see if other programmers can help tackle that. We'll keep this as a feature request but in the meantime, you can try adding another keyword to handle those synonyms.

**Comment by patapizza** [comment]
Can you share an example of what you're trying to do? Thanks.

**Comment by oliverjessner** [comment]
@patapizza We want to block words like "corona" because wit thinks most of the time it's a location or contact. 
And thinks like "So" which is an adverb in german is recognized as a datetime (Sonntag eng.: Sunday).

Does pretrained data have any impact of the intents?

**Comment by patapizza** [comment]
Thanks for the additional info.

So filtering out built-in entities would be applying band-aid. We're tracking improvements to the built-in models.
In the meantime, I'd recommend applying a blacklist filtering when you're processing Wit response (the same way you'd ignore intents/entities below a certain confidence threshold).
Thank you for reporting the issue with `wit/datetime`. It's because of this: https://github.com/facebook/duckling/blob/master/Duckling/Time/DE/Rules.hs#L58. I'm working on a fix.

The built-in entities don't have any impact on the intents.

Hope this helps.

**Comment by patapizza** [comment]
`so` should no longer trigger `wit/datetime` in German, whereas `so.` should still do.
